src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -139,6 +143,22 @@ private List<String> allowedActions(MigrationProject project) throws Exception {
         return allowedActions;
     }
 
+    public ObjectNode processingJobs(MigrationProject project, ObjectMapper mapper) throws Exception {
+        ObjectNode processingJobs = mapper.createObjectNode();
+
+        // velocicropter report status
+        ObjectNode velocicropterStatus = mapper.createObjectNode();
+        if (Files.exists(project.getProjectPath().resolve("processing/results/velocicroptor/job_completed"))) {
```

Comment:
we should use this constant for where to find the velocicroptor results, to make it easier to keep things synced:
https://github.com/UNC-Libraries/chompb/blob/main/src/main/java/edu/unc/lib/boxc/migration/cdm/jobs/VelocicroptorRemoteJob.java#L28
you can change it to public